packages/forms-web-app/src/lib/documents-api-wrapper.js

```diff
@@ -20,7 +20,12 @@ exports.createDocument = async (appeal, formData) => {
   let apiResponse;
   try {
     const fd = new FormData();
-    fd.append('file', fs.createReadStream(formData.tempFilePath), formData.name);
+
+    if (typeof data === 'object') {
+      fd.append('file', fs.createReadStream(data.tempFilePath), data.name);
+    } else {
```

Comment:
One thing I hit upon when doing that original line was that sometimes I'd get an 0 byte PDF created.

I'd imagine:

```
const data = undefined;
fd.append('file', data, `${appeal.id}.pdf`);
```

would create a 0 byte PDF. 

Maybe worth being a bit more defensive here? 

```
else if (typeof data === 'string' || data instanceof String) {
//...
} else {
throw?
}